### ISSUE TITLE
Markup for M2 output in Emacs

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2-emacs#79]
Quoting from @d-torrance in https://github.com/Macaulay2/M2/issues/3610#issuecomment-2549474475:
> > Tangentially, it would be great if M2-mode could replace strings wrapped in `$ .. $` with rudimentary tex, but this may be nontrivial.
> 
> It can! See https://github.com/Macaulay2/M2/wiki/Previewing-TeX-in-Macaulay2-Emacs-buffers

This is useful, but I think has a few drawbacks:
1. we don't want typing `print "$ blah $" in the input line to produce LaTeX
2. we don't want an output like `use the synonym Cremona$point` to produce LaTeX

And a big missing features:
3. maybe we need a new `topLevelMode` similar to `WebApp` that produces LaTeX instead of html.

**Comment by YueRen** [comment]
I tried a solution in https://github.com/Macaulay2/M2-emacs/pull/93, but it requires [`dvipng`](https://packages.debian.org/sid/dvipng) in addition to a functioning `auctex`.

**Comment by mahrud** [comment]
Fair enough, perhaps this is not super desirable, and I can think of better ways of doing this now (e.g https://github.com/Macaulay2/M2/issues/3616)

**Comment by mahrud** [comment]
Oops didn't mean to close, but fine if people think it should be closed.
